src/display_servers/xlib_display_server/xatom.rs

```diff
@@ -1,6 +1,9 @@
 use std::ffi::CString;
 use x11_dl::xlib;
 
+// Specifications can be found here:
+// https://specifications.freedesktop.org/wm-spec/1.3/ar01s03.html
```

Comment:
This is very helpful ❤️

Reply:
I'm often confused by signed and unsigned numbers from C stuff :sweat_smile: